tests/framework/installer/ceph_installer.go

```diff
@@ -1047,6 +892,17 @@ func (h *CephInstaller) waitForCleanupJobs(namespace string) error {
 			if job.Status.Failed > 0 {
 				return false, fmt.Errorf("job %s failed", job.Name)
 			}
+			if job.Status.Succeeded == 1 {
+				req := h.k8shelper.Clientset.CoreV1().Pods(namespace).GetLogs(job.Name, &v1.PodLogOptions{})
+				readCloser, err := req.Stream()
+				if err != nil {
+					return false, fmt.Errorf("failed to read from stream. %+v", err)
```

Comment:
It's not clear why we are failing to read the stream in the test. I would suggest we ignore this error and allow the test to continue. Capturing the logs is just informational and at least works sometimes so we can see what is happening. The helm test has a slightly different shutdown procedure that may be affecting this.

Reply:
The thing is nothing is happening, that's why I wanted to get the logs and see if the job was wipping devices correctly :-(